wiki/Tournaments/MCNC/2021/en.md

```diff
@@ -0,0 +1,130 @@
+---
+tags:
+  - MCNC
+  - MCNC2021
+---
+
+# osu!mania 4K Chinese National Cup 2021
+
+![MCNC 2021 logo](img/logo.png)
+
+The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the third installment of the osu!mania 4K Chinese National Cup.
```

Comment:
`The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) is a country-based osu!mania tournament hosted by the \[Crz\]Team. It is the third installment of the osu!mania 4K Chinese National Cup.`

(the tournament is still ongoing so no need to use past tense here)

Reply:
Done!